src/Calendar/MonthDropdown.js

```diff
@@ -2,19 +2,20 @@
 
 import * as React from 'react';
 import classNames from 'classnames';
-import moment from 'moment';
 import List from 'react-virtualized/dist/commonjs/List';
 import AutoSizer from 'react-virtualized/dist/commonjs/AutoSizer';
 import { prefix, getUnhandledProps, defaultProps } from '../utils';
 import MonthDropdownItem from './MonthDropdownItem';
+import { getYear, setYear, setMonth, getMonth } from 'date-fns';
+import curry from '../utils/curry';
```

Comment:
`lodash` already exists in lib, it is best to use `_.curry` directly.

Reply:
I renamed curry to composeFunctions. lodash/curry - different helper.
But usefull for code understanding on piped execution.